graalvm-ce/Dockerfile

```diff
@@ -0,0 +1,64 @@
+#
+# Scala and sbt Dockerfile
+#
+# https://github.com/hseeberger/scala-sbt
+#
+
+# Pull base image
+ARG BASE_IMAGE_TAG
+FROM oracle/graalvm-ce:${BASE_IMAGE_TAG:-19.3.0-java11}
+
+# Env variables
+ARG SCALA_VERSION
+ENV SCALA_VERSION ${SCALA_VERSION:-2.13.1}
+ARG SBT_VERSION
+ENV SBT_VERSION ${SBT_VERSION:-1.3.3}
+
+# Install sbt
+RUN \
+  curl -L -o sbt-$SBT_VERSION.rpm https://bintray.com/sbt/rpm/download_file?file_path=sbt-$SBT_VERSION.rpm && \
+  rpm -i sbt-$SBT_VERSION.rpm && \
+  rm sbt-$SBT_VERSION.rpm && \
+  yum -y update && \
+  yum -y install sbt
+
+# Add and use user sbtuser
+RUN groupadd --gid 1001 sbtuser && useradd --gid 1001 --uid 1001 sbtuser --shell /bin/bash
+RUN chown -R sbtuser:sbtuser /opt
+# RUN mkdir /home/sbtuser && chown -R sbtuser:sbtuser /home/sbtuser
+RUN mkdir /logs && chown -R sbtuser:sbtuser /logs
+USER sbtuser
+
+# Switch working directory
+WORKDIR /home/sbtuser
+
+# Install Scala
+## Piping curl directly in tar
+RUN \
+  curl -fsL https://downloads.typesafe.com/scala/$SCALA_VERSION/scala-$SCALA_VERSION.tgz | tar xfz - -C /home/sbtuser/ && \
+  echo >> /home/sbtuser/.bashrc && \
+  echo "export PATH=~/scala-$SCALA_VERSION/bin:$PATH" >> /home/sbtuser/.bashrc
+
+# Prepare sbt
+RUN \
+  sbt sbtVersion && \
+  mkdir -p project && \
+  echo "scalaVersion := \"${SCALA_VERSION}\"" > build.sbt && \
+  echo "sbt.version=${SBT_VERSION}" > project/build.properties && \
+  echo "case object Temp" > Temp.scala && \
+  sbt compile && \
+  rm -r project && rm build.sbt && rm Temp.scala && rm -r target
+
+# Link everything into root as well
+# This allows users of this container to choose, whether they want to run the container as sbtuser (non-root) or as root
+USER root
+RUN \
+  echo "export PATH=/home/sbtuser/scala-$SCALA_VERSION/bin:$PATH" >> /root/.bashrc && \
+  ln -s /home/sbtuser/.ivy2 /root/.ivy2 && \
```

Comment:
Looks like another dockerfiles has this line 
ln -s /home/sbtuser/.cache /root/.cache && \
but I'm not sure that it is needed

Reply:
good point, going to sync with the oracle image